src/binary-reader-objdump.c

```diff
@@ -65,34 +67,40 @@ static void WASM_PRINTF_FORMAT(2, 3)
   va_end(args);
 }
 
-static WasmResult do_begin_section(Context* ctx,
-                                   const char* name,
-                                   size_t offset,
-                                   size_t size) {
-  WasmBool section_match = !ctx->options->section_name ||
-                           !strcasecmp(ctx->options->section_name, name);
+static WasmResult begin_section(WasmBinaryReaderContext* ctx,
```

Comment:
I agree it should be consistent but almost all the WasmBinaryReaderContext in this file are currently called 'ctx'.   Perhaps a cleanup CL should follow